### ISSUE TITLE
sid_msg: syntaxtically correct rules not parsed properly

### ISSUE THREAD

**Issue by GoogleCodeExporter** [ype/pulledpork#67]
```
any rule which has whitespace after the '(' will get the whitespace prepended 
to the first tag (normally msg:) thus the tag will not be recognised...

Version 5.0

fix -- add ' *' after the \( at end of re.  This strips any white space after 
the parenthesis.

( my $header, my $options ) =
              split( /^[^"]* \( */, $$ruleshash{$k}{$k2}{'rule'} )                                                                                                                                                                 


```

Original issue reported on code.google.com by `russell....@gmail.com` on 7 Mar 2011 at 12:17

**Comment by GoogleCodeExporter** [comment]
Original comment by `Cummin...@gmail.com` on 7 Mar 2011 at 2:31
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Can you provide a sample sid for review?
```

Original comment by `Cummin...@gmail.com` on 7 Mar 2011 at 2:55

**Comment by GoogleCodeExporter** [comment]
```
any rule with white space after the opening parenthesis:

here is one of my automatically produced rules from a IP black list:

alert tcp $HOME_NET any -> 203.222.xx.yy 6667 ( msg: "Botnet CandC 
irc.xxxx.zz"; 
tag: session,20,packets; classtype:trojan-activity; sid: 9000000; rev: 1;)

sanitized since the list is not public...

BTW probably best to sustitute \s for the ' ' I added that will cope with tabs, 
which I believe are legal (although I can't see anyone putting a tab at the 
start....)
```

Original comment by `russell....@gmail.com` on 7 Mar 2011 at 7:47

**Comment by GoogleCodeExporter** [comment]
```
Committed fix rev #219
```

Original comment by `Cummin...@gmail.com` on 11 Mar 2011 at 5:57
- Changed state: **Fixed**
